model-optimizer/extensions/middle/AddMeanScaleValues.py

```diff
@@ -64,7 +64,14 @@ def insert_pre_processing(graph: Graph, input_node: Node, node_mean_scale_values
 
         for dst in input_node.out_port(0).get_destinations():
             if dst.node.soft_get('type') != 'ShapeOf':
+                # After the insertion of additional operations model optimizer
+                # should keep the link to the input layer. Parameter node in framework
+                # should map to parameter node in IR.
+                # For this reason 'fw_tensor_debug_info' should be kept in data node.
+                tmp = input_node.out_node(0)['fw_tensor_debug_info']
```

Comment:
could you please rename `tmp` variable because now it sounds as artifact remained after debugging:)